src/Debugger/Expando.elm

```diff
@@ -14,19 +14,33 @@ import Html exposing (Html, div, span, text)
 import Html.Attributes exposing (class, style)
 import Html.Events exposing (onClick)
 import Json.Decode as Json
+import Set exposing (Set)
 
 
 
 -- MODEL
 
 
-type Expando
+type alias Expando =
+  { unexpanded : Unexpanded
+  , expanded : Set Path
+  , viewMore : Dict Path Int
+  }
+
+
+type alias Path = List String
+
+
+type Unexpanded = Unexpanded
```

Comment:
I'm not sure I understand the value of this type, drive it doesn't hold any data 🤔